tensorrt_llm/_torch/models/modeling_phi4mm.py

```
@@ -332,6 +330,8 @@ def dynamic_preprocess(
 
     Ref code: https://huggingface.co/microsoft/Phi-4-multimodal-instruct/blob/main/processing_phi4mm.py#L201
     """
+    import torchvision
```

Flake8 (7.3.0)

[error] 'torchvision' imported but unused

(F401)